cli/src/commands/new/index.ts

```diff
@@ -60,6 +52,7 @@ export default class NewCommand extends Command {
     sdk: Flags.string({
       char: "s",
       description: "SDK to use",
+      options: ["go", "assemblyscript"],
```

Comment:
Let's allow `"as"` and `"golang"` also.  The `parseSDK` function can accept these already.